backend/utils/nunjucksSetup.js

```diff
@@ -40,6 +40,21 @@ module.exports = (app, path) => {
     ]
   })
 
+  njkEnv.addFilter('addDefaultSelectedVale', (items, text) => {
```

Comment:
Going to get some unit tests in for my filters in a later ticket

Reply:
meh.. typo in your filter